ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -652,6 +654,7 @@ module Vendor_nvidia = struct
             )
           in
           let devid = find_one_by_name "devId" vgpu_type in
+          let vclass = get_attr "class" vgpu_type in
```

Comment:
Can we rely on the `class` attribute to be always there including in old XML files?

Reply:
We are not going to backport this and from what I see this attribute is present in the current nvidia drivers.